src/machines/modelingMachine.ts

```diff
@@ -736,6 +748,62 @@ export const modelingMachine = setup({
       // eslint-disable-next-line @typescript-eslint/no-floating-promises
       codeManager.updateEditorWithAstAndWriteToFile(kclManager.ast)
     },
+    'AST offset plane': ({ event }) => {
+      if (event.type !== 'Offset plane') return
+      if (!event.data) return
+      ;(async () => {
```

Comment:
I know you're folowing existing patterns, but maybe this action should be an invoked service (or whatever the v5 naming is now)
![image](https://github.com/user-attachments/assets/b349d28c-7881-4369-b457-3cb2224b4c12)

I think this should still tie into the command bar okay because it should still just need to send a single event yeah?

That way we don't need to jam an `;(async () => {...` into an action.

Reply:
Great idea, done